CryptographyCodingExerciseWeek13/CryptographyCodingExerciseWeek13/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Security.Permissions;
+using System.Security.Principal;
+using System.Text;
+
+namespace CryptographyCodingExerciseWeek13
+{
+    class Program
```

Comment:
For a coding exercise like this. you can create a class library rather than a console application. A class library has no Main method entry point and relies on another program to utilize its code, such as your test project.